### DISCUSSION TITLE
Documentation needs love :heart:

### DISCUSSION THREAD

**Discussion by project0** [argoproj/argo-cd#21104] [General]
I am a quiet new user to argocd (used fluxcd in the past) and i have really trouble to navigate through the documentation to find the relevant information i am looking for. 
It is not even the content itself, the menu/navigation entries are confusing and contains random sorted things that makes it super hard to find anything. Especially new users get lost very quickly. 
Also the separation between user and operator is sometimes very hard and vague to me, i dont see what makes the difference here (I guess with users (developers) its mostly directed towards "resource manifests writers"?). Splitting the docs based on personas is not really helpful in my opinion. 

Can we do something to refactor the sections and move documentation in contextual places so its much easier to read and kinda guided? 

I could imagine some AI magic would already help to do so. 
wdyt?

**Comment by pasha-codefresh** [comment]
It would be great if you open Proposal with ideas how you would restructure it. I think @revitalbarletz may help you to do it , if you wish

**Comment by kostis-codefresh** [comment]
> Can we do something to refactor the sections and move documentation in contextual places so its much easier to read and kinda guided?

@project0 You might find this interesting as well https://github.com/argoproj/argo-cd/issues/5635
